static/src/javascripts-legacy/projects/common/modules/video/video-container.js

```diff
@@ -5,29 +5,61 @@ define([
     'lib/element-inview',
     'bootstraps/enhanced/media/video-player',
     'lodash/objects/assign',
-    'lib/create-store'
+    'lib/create-store',
+    'common/modules/atoms/youtube',
+    'lib/detect'
 ], function (
     bean,
     fastdom,
     $,
     ElementInview,
     videojs,
     assign,
-    createStore
-) {
+    createStore,
+    youtube,
+    detect
+)
+{
+    function updateYouTubeVideo(currentItem){
+        var youTubeAtom = currentItem.querySelector('.youtube-media-atom');
+        if(youTubeAtom) {
+            return youtube.onVideoContainerNavigation(youTubeAtom.dataset.uniqueAtomId);
```

Comment:
`.dataset` isn't available [<IE11](http://caniuse.com/#search=dataset), might want to use bonzo or `getAttribute` for native.